web_poet/serialization/functions.py

```diff
@@ -193,3 +196,32 @@ def _deserialize_Stats(cls: Type[Stats], data: SerializedLeafData) -> Stats:
 
 
 register_serialization(_serialize_Stats, _deserialize_Stats)
+
+
+def _serialize_AnnotatedResult(o: AnnotatedResult) -> SerializedLeafData:
+    serialized_data: SerializedLeafData = {
+        "metadata.json": json.dumps(o.metadata).encode(),
```

Comment:
Let's use some common json options, like indent and ensure_ascii, to make this file more readable.